core-java-modules/pre-jpms/pom.xml

```diff
@@ -72,8 +80,8 @@
     <properties>
         <dependency.plugin.version>3.1.1</dependency.plugin.version>
         <compiler.plugin.version>3.8.0</compiler.plugin.version>
-        <source.version>1.8</source.version>
-        <target.version>1.8</target.version>
+        <source.version>1.9</source.version>
```

Comment:
let's upgrade to JDK11 directly as we are doing for all other modules

Reply:
Done